content/gjtempleton/drone-pypi/index.md

```diff
@@ -0,0 +1,58 @@
+---
+date: 2018-02-11T00:00:00+00:00
+title: Pypi
+author: gjtempleton
+tags: [ python, pypi ]
+repo: gjtempleton/drone-pypi
+logo: pypi.svg
+image: gjtempleton/drone-pypi
+---
+
+This plugin allows you to publish Python pypi packages as part of your pipeline.
+
+Basic example:
+
+```yaml
+pipeline:
+  pypi_publish:
+    image: gjtempleton/drone-pypi
+    username: guido
+    password: secret
+```
+
+This block will first generate the relevant .pypirc file with the provided details generating a _repo_ block, then execute the following command:
+
+```
+python3 setup.py sdist upload -r repo
+```
+
+You can also specify the distribution types you want to upload, as well as the repository URL and relative path of the setup python file.
+```diff
+pipeline:
+    pypi_publish:
```

Comment:
Can you unindent for the rest of the examples as well?
Thanks!

Reply:
Whoops, serves me right for pushing changes in the midst of a meetup. Think that's them all now.